spec/src/main/asciidoc/required-metrics.adoc

```diff
@@ -251,3 +251,57 @@ Note: This is a vendor specific attribute/operation that is not defined in java.
 |MBean| java.lang:type=OperatingSystem (com.sun.management.UnixOperatingSystemMXBean for Oracle Java, similar one exists for IBM Java: com.ibm.lang.management.ExtendedOperatingSystem)
 Note: This is a vendor specific attribute/operation that is not defined in java.lang
 |===
+
+
+=== REST
+
+The MicroProfile Metrics runtime will track REST metrics from RESTful resources whenever they are utilized in the application. It is up to the implementation to decide how to enable the REST metrics.
+These metrics require that RESTful stats are gathered from all RESTful resources.
+
+*RESTRequests*
+[cols="1,4"]
+|===
+|Name| rest.request
+|DisplayName| Total Requests and Response Time
+|Type| SimpleTimer
+|Unit| None
```

Comment:
We should have a unit defined here, because it is a timer.
What would it be? Milliseconds  are maybe not precise enough. Nanoseconds?

Reply:
I think that only matters currently for our JSON output, but I'd agree with nanos.

Reply:
nanos matches with the default unit used by `@Timed`, which I think makes it a good choice.  Note that when the metric is created people can specify a different unit (eg. ms), which we should be taking into account.